lib/pure/collections/chains.nim

```diff
@@ -1,44 +0,0 @@
-#
-#
-#            Nim's Runtime Library
-#        (c) Copyright 2016 Andreas Rumpf
-#
-#    See the file "copying.txt", included in this
-#    distribution, for details about the copyright.
-#
-
-## Template based implementation of singly and doubly linked lists.
-## The involved types should have 'prev' or 'next' fields and the
-## list header should have 'head' or 'tail' fields.
-
```

Comment:
I like this module so keep it. Besides, we have a deprecation period.

Reply:
I don't think we need a deprecation period here. This module is entirely useless. Nobody uses it, nor is it helpful for anybody or anything. This module takes up one entry at the very top of [The Index](https://nim-lang.org/docs/theindex.html) that leads to a disappointment. Removing this module improves the index without breaking anything. If you like this module personally, you should move it personal project.

Reply:
Remove it from the produced docs then, but don't remove this module, mark it deprecated.

Reply:
I do I exclude a module from documentation generation? Is there a module that is already excluded, where I can look up how to do it?

Reply:
kochdocs.nim ``withoutIndex``